scheduling-api-graphql/scheduling-api-graphql-beans/scheduling-api-graphql-beans-input/src/main/java/org/ow2/proactive/scheduling/api/graphql/beans/input/JobInput.java

```diff
@@ -50,6 +50,10 @@ private JobInput(String queryString) {
 
         private String beforeSubmittedTime;
 
+        private String greaterThanId;
```

Comment:
yes, we use input as the filter in graphql, all classes suffixed with input is supposed to be used as a filter wrapper class